libs/core/src/lib/dialog/dialog-service/dialog.service.ts

```diff
@@ -5,6 +5,8 @@ import { DynamicComponentService } from '../../utils/dynamic-component/dynamic-c
 import { DialogRef } from '../utils/dialog-ref.class';
 import { DialogBaseService } from '../base/dialog-base.service';
 import { DialogDefaultContent } from '../utils/dialog-default-content.class';
+import { RtlService } from '../../utils/public_api';
```

Comment:
please prefer reference to the direct entity instead of a barrel import, in some cases it causes build issues